hw/ip/spi_host/doc/theory_of_operation.md

```diff
@@ -616,61 +643,66 @@ This can be achieved by also setting the [`CONTROL.OUTPUT_EN`](registers.md#cont
 
 ### Component reset
 
-In addition to the global hardware reset, there is a software reset option which completely resets the SPI host.
+In addition to the global hardware reset, there is a software reset option which completely resets the SPI_HOST IP.
 To use this reset, assert [`CONTROL.SW_RST`](registers.md#control), and then wait for the device to reset ([`STATUS.ACTIVE`](registers.md#status), [`STATUS.TXQD`](registers.md#status) and [`STATUS.RXQD`](registers.md#status) to all go to zero), before releasing [`CONTROL.SW_RST`](registers.md#control).
 
-## Block Diagram
+# Design Details
+
+## Block Diagrams
 
 ![](../doc/spi_host_block_diagram.svg)
 
-# Design Details
+![](../doc/spi_host_passthrough.svg)
```

Comment:
Would it make sense to pull this line into the commit that split out the diagrams, to avoid an orphan SVG for a couple of commits? (Don't bother if it's lots of work!)

Reply:
Yep done :+1: